var/spack/repos/builtin/packages/r-jquerylib/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RJquerylib(RPackage):
+    """Obtain 'jQuery' as an HTML Dependency Object."""
+
+    homepage = "https://CRAN.R-project.org/package=jquerylib"
```

Comment:
You can drop this as the project page is part of the `cran` attribute.

Reply:
Done. Thanks for the hint.